### ISSUE TITLE
Improve accessibility of system

### ISSUE THREAD

**Issue by arbron** [koboldpress/black-flag#338]
- Ensure are controls that are just icons have the appropriate `aria-labels` or `title` attributes
- Make sure everything that is intractable is properly intractable and tabable
  - Either use buttons
  - Or support interacting with links using "Enter" and "Spacebar" keys
- Verify contrast
- Verify legibility in various colorblindness modes
